spec/meilisearch/client/tasks_spec.rb

```diff
@@ -50,16 +50,35 @@
     expect(tasks['next']).to be_a(Integer)
   end
 
-  it 'filters tasks with index_uid/type/status' do
-    tasks = client.tasks(index_uid: ['a-cool-index-name'])
+  it 'filters tasks with index_uidx/typex/statuses' do
```

Comment:
Is it intended to have `typex` instead of `types`, and `index_uidx` instead of `index_uids`?

Reply:
No, it was a mistake, thanks for catching that! :D